packages/nimble-components/src/patterns/anchor/tests/anchor-patterns.stories.ts

```diff
@@ -0,0 +1,157 @@
+import type { Meta, StoryObj } from '@storybook/html';
+import { html, ref } from '@microsoft/fast-element';
+import { createUserSelectedThemeStory } from '../../../utilities/tests/storybook';
+import {
+    controlLabelFont,
+    controlLabelFontColor,
+    mediumPadding,
+    standardPadding
+} from '../../../theme-provider/design-tokens';
+import { anchorTag } from '../../../anchor';
+import { anchorButtonTag } from '../../../anchor-button';
+import { anchorTabsTag } from '../../../anchor-tabs';
+import { anchorTabTag } from '../../../anchor-tab';
+import { breadcrumbTag } from '../../../breadcrumb';
+import { breadcrumbItemTag } from '../../../breadcrumb-item';
+import { RichTextViewer, richTextViewerTag } from '../../../rich-text/viewer';
+import { anchorTreeItemTag } from '../../../anchor-tree-item';
+import { treeViewTag } from '../../../tree-view';
+import { anchorMenuItemTag } from '../../../anchor-menu-item';
+import { menuTag } from '../../../menu';
+import { Table, tableTag } from '../../../table';
+import { tableColumnAnchorTag } from '../../../table-column/anchor';
+
+const hrefDescription = 'To disable the control, remove the `href` attribute.';
+
+interface AnchorPatternsArgs {
+    label: string;
+    href: string;
+    tableRef: Table;
+    setTableData: (args: AnchorPatternsArgs) => void;
+    richTextViewerRef: RichTextViewer;
+    setRichTextViewerData: (args: AnchorPatternsArgs) => void;
+}
+
+const metadata: Meta<AnchorPatternsArgs> = {
+    title: 'Tests/Anchor Patterns',
+    parameters: {
+        actions: {}
+    },
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html`
+        <style class='code-hide'>
+            .control-container {
+                margin: var(${standardPadding.cssCustomProperty});
+            }
+
+            .label {
+                font: var(${controlLabelFont.cssCustomProperty});
+                color: var(${controlLabelFontColor.cssCustomProperty});
+                margin-bottom: var(${mediumPadding.cssCustomProperty});
+            }
+        </style>
+        <div class="control-container">
+            <div class="label">Native anchor</div>
+            <a href="${x => x.href}">${x => x.label}</a>
+        </div>
+
+        <div class="control-container">
+            <div class="label">${anchorTag}</div>
+            <${anchorTag} href="${x => x.href}">${x => x.label}</${anchorTag}>
+        </div>
+
+        <div class="control-container">
+            <div class="label">${anchorButtonTag}</div>
+            <${anchorButtonTag} href="${x => x.href}">${x => x.label}</${anchorButtonTag}>
```

Comment:
Didn't check all controls but i.e. the anchor button is disabled via the disabled attribute instead of an empty href

Reply:
I've updated the story to not have the `href` configurable. Instead, I've added a `disabled` configuration, and each component uses that appropriately to either set `disabled` on the component or set the component's `href` to `undefined`.

The one minor exception is the rich text viewer. It doesn't have a concept of a disabled anchor or an anchor without an href, so I made it set the link to an empty string, which renders as `<>`.